src/main/java/core/basesyntax/service/writer/WriteFileFruitImpl.java

```
@@ -0,0 +1,22 @@
+package core.basesyntax.service.writer;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteFileFruitImpl implements WriteFileFruit {
+    @Override
+    public void writeAll(String string, String path) {
+        if (string == null || string.isBlank()) {
+            throw new RuntimeException("Output content must not be null");

```

The validation message `"Output content must not be null"` is inconsistent with the check which rejects null *and* blank values. Make the message explicit and informative (e.g., `"Output content must not be null or blank"`). This aligns with the checklist requirement to throw informative RuntimeExceptions: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an **informative** message and exception object."